README.md

```diff
@@ -95,7 +95,9 @@ There are currently 3 different sets of tests:
 * Production tests, that run in node.js, but which interact with external services.
 
 The `npm test` command will run the browser and Node based tests.  Please note, the node tests may behave oddly if you
-have set the `NODE_ENV` variable.
+have set the `NODE_ENV` variable.  The last two types of tests require you to have either Docker or Vagrant installed
```

Comment:
Can you add material at top level here explaining the use of the GPII_TEST_COUCH_USE_VAGRANT env var

Reply:
OK, I added a bit more detail.